### PR TITLE
Cleanup of src/qt/forms/aboutdialog.ui

### PR DESCRIPTION
**Pull request by mitchellcash**
Initial cleanup of aboutdialog.ui. Further changes to this in mind I just want to get this base cleanup done and I'll have further pulls to follow.

### CODE REVIEW

**Comment by mitchellcash** [comment]
Rebased to include translation file fixes. Didn't actually look into the correctness of the translations just moved around to fit the change.

File: src/qt/forms/aboutdialog.ui [hunk 1]
```diff
@@ -81,8 +59,11 @@
        </item>
        <item>
         <widget class="QLabel" name="versionLabel">
+         <property name="cursor">
+          <cursorShape>IBeamCursor</cursorShape>
+         </property>
          <property name="text">
-          <string notr="true">v0.1.0</string>
+          <string notr="true">0.3.666-beta</string>
```
Comment (IngCr3at1on) [inline src/qt/forms/aboutdialog.ui]: What is this version?

**Comment by IngCr3at1on** [comment]
Builds and otherwise looks good (the about dialog actually fits in the window for a change).

I also have not confirmed validity of the translations.

File: src/qt/forms/aboutdialog.ui (hunk 1 above)
Comment (mitchellcash) [inline src/qt/forms/aboutdialog.ui]: It's actually just used as preview text in Qt Designer.

I can change it back if needed but this seemed more appropriate. I just realised though I could have improved it yet again with something like 0.3.6.9-beta but to be honest it doesn't actually effect anything in production so I don't mind.

**Comment by mitchellcash** [comment]
@IngCr3at1on my favourite part is it now fitting!

All translations need some work but that's not really the point of this pull :)

**Comment by IngCr3at1on** [comment]
Yeah I'm fine with this, gets my vote.

**Pull request merged**